source/includes/_widgets.md

```diff
@@ -204,6 +204,94 @@ to                  | End date of the tracking period
 participation_ratio | Employee participation ratio in percent
 happiness_score     | Average happiness score for the tracking period
 
+## Company participation
+
+```http
+GET /widgets/company-participation HTTP/1.1
+Host: api.company-mood.com
+Content-Type: application/json
+Accept: application/vnd.company-mood-v2+json
+Authorization: Bearer 795665b4-53da-468c-a0d7-ab2d82e58406
+X-App-Token: 27f50875-9a43-4d6c-a376-6968f09858db
+```
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+  "data": {
+    "id": "company-participation",
+    "type": "widgets",
+    "attributes": {
+      "moods_count": 2,
+      "employee_amount": 10,
+      "participation_in_percent": 12,
+      "moods_in_percent": [
+        { "feeling": "sad", "percent": 50 },
```

Comment:
I would do it a bit different. 

For example:
```
{
  moods_details: [
    { "feeling": "happy", "percent": 50, "count": 5 }, //...
  ]
}
```

This way, we can add details and also count, etc. 
What do you think?

Reply:
Good idea. I'll change it asap 👍